src/core/fbs/mutator/fetcher.ts

```diff
@@ -1,4 +1,4 @@
-import { getToken, TOKEN_USER_KEY } from "../../token";
+import { getToken, TOKEN_LIBRARY_KEY, TOKEN_USER_KEY } from "../../token";
```

Comment:
@spaceo what about these GitHub warnings?
'TOKEN_USER_KEY' is defined but never used

Reply:
👍